### ISSUE TITLE
ARIA support

### ISSUE THREAD

**Issue by davidtmnolan** [vitalets/combodate#80]
Are there any plans to add support for ARIA attributes on the individual drop downs?  Since combo data is one input in the markup I cannot add accessibility attributes to each individual year-month-day dropdown.  The only solution I have is to use JS.  Are there any other solutions?
